addons/mail/static/src/core_ui/message.scss

```diff
@@ -18,6 +18,10 @@
 
 .o-mail-Message-body > p {
     margin-bottom: 0;
+
+    &:last-of-type {
+        display: inline-block;
+    }
```

Comment:
`float` was breaking style of html content, so I kept t-outing body and just made last paragraph inline so (edited) label is inline.